sdk/core/core-rest-pipeline/src/createPipelineFromOptions.ts

```diff
@@ -95,7 +95,7 @@ export function createPipelineFromOptions(options: InternalPipelineOptions): Pip
   pipeline.addPolicy(multipartPolicy(), { afterPhase: "Deserialize" });
   pipeline.addPolicy(defaultRetryPolicy(options.retryOptions), { phase: "Retry" });
   pipeline.addPolicy(tracingPolicy(options.userAgentOptions), { afterPhase: "Retry" });
-  if (isNode) {
+  if (isNodeLike) {
```

Comment:
@mpodwysocki don't Deno and Bun use fetch instead of node's http module?

Reply:
Both have support for the Node http module ([Deno](https://docs.deno.com/runtime/manual/node/compatibility), [Bun](https://bun.sh/docs/runtime/nodejs-apis)) in addition to fetch. I don't know which one we should prefer.